### DISCUSSION TITLE
> Task :react-native-firebase_firestore:compileDebugJavaWithJavac FAILED

### DISCUSSION THREAD

**Discussion by britny-no** [invertase/react-native-firebase#4408] [Q&A] [answered]
when i install native-firebase/firestore, and run command "npm run android", under error appear
"> Task :react-native-firebase_firestore:compileDebugJavaWithJavac FAILED"

somebody help me!!!! is there setting added some code in build.gralde file??

**Comment by mikehardy** [comment, marked as answer]
Hi there! Failed why? There is no actionable information here, sorry.

You can use this to generate a skeletal project that works, if you read the comments in the script you will see what it's doing so you can reproduce success in your project https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh

**Comment by britny-no** [comment]
oh thank you let me try~


2020년 10월 17일 (토) 오전 5:24, Mike Hardy <notifications@github.com>님이 작성:

**Reply by andersonaddo** [reply]
Judging from the silence, I take it that you found the solution
